packages/glasscoder/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: this.ven <misgi@ven.uber.space>
+
+pkgname=glasscoder
+pkgver=1.0.4
+pkgrel=1
+pkgdesc="Minimalist audio encoder for IceCast and Shoutcast streaming
+servers as well as HTTP Live Streaming (HLS)"
+arch=(x86_64)
+url="https://github.com/ElvishArtisan/GlassCoder"
+license=(GPL2)
+depends=(alsa-lib jack2 curl lame libfdk-aac libogg libsamplerate libsndfile libvorbis openssl opus python qt5-tools taglib twolame)
+makedepends=(docbook-xsl docbook5-xml libxslt)
+source=(https://github.com/ElvishArtisan/GlassCoder/releases/download/v1.0.4/glasscoder-1.0.4.tar.gz)
+sha256sums=('350714c1cf23caf1e2f12431385a946ef8ca4a585329a399cebcf5ad598ee7c8')
+
+build() {
+  export DOCBOOK_STYLESHEETS="/usr/share/xml/docbook/xsl-stylesheets-1.79.2"
```

Comment:
put the this line directly before the `:/configure` line.

Reply:
Isn't it also bad practice to `export` environment variables in a PKGBUILD? Not 100 % sure about this now but maybe we should do
```
DOCBOOK_STYLESHEETS="..." ./configure ...

Reply:
Unfurtunately that doesn't work. Compilation fails because of missing variable.

Reply:
Probably it is also needed in the environment of `make`. `export`ing in the `PKGBUILD` is fine, IMO. It only affects the shell context of `makepkg` and sub-processes.

Reply:
Ok, I just remember reading that somewhere but I agree with you :D